docs/samples/ohlc.html

```diff
@@ -0,0 +1,103 @@
+<!DOCTYPE html>
+<html lang="en-US">
+<head>
+	<meta charset="utf-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1">
+	<title>Financial > OHLC | Chart.js sample</title>
+	<script src="../moment.js" type="text/javascript"></script>
+	<script src="../Chart.bundle.js" type="text/javascript"></script>
+	<script src="../Chart.Financial.js" type="text/javascript"></script>
+	<link rel="stylesheet" type="text/css" href="../style.css">
+	<link rel="icon" href="../favicon.ico"/>
+</head>
+<body>
+
+	<a href="../index.html">Home</a>
+	<div class="content">
+		<h2>OHLC</h2>
+		<div style="width:1000px">
+			<canvas id="chart1"></canvas>
+		<div>
+		<h2>Custom OHLC</h2>
+		<div style="width:1000px">
+			<canvas id="chart2"></canvas>
+		<div>
+	</div>
+
+	<script>
+		function randomNumber(min, max) {
```

Comment:
we should also move these into a common js file rather than copy/pasting between samples. perhaps create a function called `generateSampleData` or something similar

Reply:
Moved to `utils.js`